### ISSUE TITLE
Error in a11y

### ISSUE THREAD

**Issue by ricardomoro** [ewels/cpt-bootstrap-carousel#84]
This error appears in the  accessibility audit tool **a11y**:

    ✖ The purpose of each link should be clear from the link text

    #cptbc_973 > .left.carousel-control
    #cptbc_973 > .right.carousel-control

I'm using _Twitter Bootstrap Version 3_ output.

**Comment by ewels** [comment]
Thanks for the report - what would fix it? I notice that the [bootstrap docs](http://getbootstrap.com/javascript/#carousel) now have `role="button"` which isn't in the markup produced by the plugin.. (also screen reader only text)

**Comment by ricardomoro** [comment]
I think that besides the `role="button"`, the links need a text content. As you said, a screen reader only text should be sufficient.
Furthermore, `aria-hidden="true"` must be placed in the glyphicon `span`, such as the Bootstrap docs shows.

**Comment by ewels** [comment]
Thanks @ricardomoro - I will add these when I next do some work on the plugin (could be while, I haven't actually used it myself for years now). Feel free to open a PR if you'd like to see it added sooner.

Phil

**Comment by ricardomoro** [comment]
Ok @ewels, thank you!
I'll do that, as soon as I use the plugin again.
